<launch>
<node name="my_node_name" pkg="se306_example" type="R0" output="screen"/>

<node name="GodNode" pkg="se306_example" type="GodNode" output="screen"/>

<node name="SheepOne" pkg="se306_example" type="SheepOne" output="screen"/>

<node name="GrassOne" pkg="se306_example" type="GrassOne" output="screen"/>

<node name="Truck" pkg="se306_example" type="Truck" output="screen"/>

<node name="FarmerOne" pkg="se306_example" type="FarmerNew" output="screen"/>

</launch>
